beetmoverscript/templates/thunderbird_nightly.yml

```diff
@@ -221,6 +221,12 @@ mapping:
       destinations:
         - {{ upload_date }}-{{ branch }}/thunderbird-{{ version }}.{{ locale }}.{{ platform }}.zip
         - latest-{{ branch }}/thunderbird-{{ version }}.{{ locale }}.{{ platform }}.zip
+    target.installer.msi:
+      s3_key: {{ upload_date }}-{{ branch }}/thunderbird-{{ version }}.{{ locale }}.{{ platform }}.installer.msi
```

Comment:
This should simply be named: `installer.msi`. (The `s3_key` entry is what we'll be shown in the http://archive.mozilla.org/pub/thunderbird/candidates/65.0b3-candidates/build1/SHA256SUMS files as identifiers. No need to have the whole thing.

Reply:
Done.